PyInstaller/hooks/hook-faker.py

```diff
@@ -0,0 +1,3 @@
+from PyInstaller.utils.hooks import collect_submodules, collect_data_files
+hiddenimports = collect_submodules('faker')
+datas = collect_data_files('faker', include_py_files=True)
```

Comment:
Why is there a need to include all .py-files, too? Please show the code in the source. Thanks